sci-chemistry/tinker/tinker-8.2.1-r1.ebuild

```diff
@@ -39,13 +41,13 @@ pkg_setup() {
 	[[ ${MERGE_TYPE} != binary ]] && tc-check-openmp
 
 	fortran-2_pkg_setup
-	java-pkg-opt-2_pkg_setup
 }
 
 src_prepare() {
 	sed 's:strip:true:g' -i ../make/Makefile || die
 	[[ $(tc-getFC) =~ "ifort" ]] && eapply "${FILESDIR}"/${PV}-openmp.patch
 	default
+	java-pkg-2_src_prepare
 }
 
 src_compile() {
```

Comment:
please look at all assignments in the next functions, and add them to `local` declaration.

Reply:
> please look at all assignments in the next functions, and add them to `local` declaration.

what exactly are you asking me to localize? sorry, not clear for me :-(

Reply:
> what exactly are you asking me to localize? sorry, not clear for me :-(

thinkgs such as `i`, `_omplib`, `_fftwlib`. The idea is that every variable name used in the phase funcitons that shouldn't be set outside the phase, should be `local` to no pollute the environment.

Reply:
> > what exactly are you asking me to localize? sorry, not clear for me :-(
> 
> thinkgs such as `i`, `_omplib`, `_fftwlib`, `test`. The idea is that every variable name used in the phase funcitons that shouldn't be set outside the phase, should be `local` to no pollute the environment.

done, if not forgotten something